base/src/main/java/io/spine/validate/FieldValidator.java

```diff
@@ -102,6 +102,24 @@ protected FieldValidator(FieldDescriptor descr,
     }
 
     /**
+     * Checks if the file is for commands.
+     *
+     * @param file a descriptor of a {@code .proto} file to check
+     * @return {@code true} if the file name ends with {@code "commands"},
+     * {@code false} otherwise
+     */
+    static boolean isCommandsFile(FileDescriptor file) {
```

Comment:
I don't understand why this method is here. I.e.  `FieldValidator.isCommandsFile(..)` looks really weird.